plugins/fix-visibility/plugin.go

```diff
@@ -172,6 +173,10 @@ func (plugin *FixVisibilityPlugin) PostRunHook(
 	return plugin.PostBuildHook(isInteractiveMode, promptRunner)
 }
 
+func (plugin *FixVisibilityPlugin) CustomCommands() ([]*aspectplugin.Command, error) {
```

Comment:
Do we have to implement it even if we have none? To make it fully implement the `Plugin` interface? :/

Reply:
I think. I am not sure there is a way around this. The plugins need to implement all the other hooks even if they dont need them

Reply:
IMO we're going to have to think about that...

Reply:
I agree that its not ideal. I just dont know how we could implement it differently. Might be worth getting some input from @f0rmiga / discussing it in our planning meeting

Reply:
https://github.com/aspect-build/aspect-cli/pull/140

Reply:
I have remove this now. Thanks @f0rmiga !